lib/elixir/lib/port.ex

```diff
@@ -58,21 +92,33 @@ defmodule Port do
   end
 
   @doc """
-  See http://www.erlang.org/doc/man/erlang.html#port_info-1.
+  Returns information about the `port`
+  or `nil` if the port is closed.
 
-  Inlined by the compiler.
+  See http://www.erlang.org/doc/man/erlang.html#port_info-1.
   """
   def info(port) do
-    :erlang.port_info(port)
+    nillify :erlang.port_info(port)
```

Comment:
@josevalim please note that this is a breaking change (same for `info/2`).
Would you like me to extract it to the separate commit?